.travis.yml

```diff
@@ -14,8 +14,14 @@ addons:
       - debian-sid
     packages:
       - shellcheck
-install: tests/install.sh
-script: tests/${TEST}
+install:
+  - tests/install.sh
+  - pip install pytest-cov
+script:
+  - tests/${TEST}
+  - py.test --cov=cinch/bin --cov-config .coveragerc --cov-report term --cov-report xml --cov-report html tests
+after_success:
+  - pip install codecov && codecov
```

Comment:
This should probably live inside of the new shell script (with the pip install in the install.sh), as well. Just make sure that you have "set -e" in the script so that this only runs when the tests are successful.